### DISCUSSION TITLE
Unable to filter the PnP Search result web part using ItemMatchingcontentType

### DISCUSSION THREAD

**Discussion by mvcsharepointdev** [microsoft-search/pnp-modern-search#1486] [Q&A]
I have added the PnP Modern Search v4 web part inside our SharePoint Online tenant from this url @ https://github.com/microsoft-search/pnp-modern-search/releases/tag/4.4.1 .

Then i tried to show items from different site collections, where those items share a content type, as follow:-

https://i.stack.imgur.com/kQLYv.png

and the following refiner:-

https://i.stack.imgur.com/Edfcs.png

but i did not get any results inside the web part.. any advice?

**Comment by wobba** [comment]
You don't share how `ItemMatchingcontentType` is set up. And v4 requires you to add the guid/id of the result source, not the name.
